File: ControlRemote/Web/Dto/ActionSortByHourTimeModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Web.Dto
{
    public class ActionSortByHourTimeModel
    {
        public DateTime HourTimeAction { get; set; }
        public List<ActionPointAtTimeModel> Commands { get; set; }
    }
}

File: ControlRemote/Web/ClientApp/src/app/Dto/ActionSortByHourTimeModel.ts
import { ActionPointAtTimeModel } from "./ActionPointAtTimeModel";

export class ActionSortByHourTimeModel {
    hourTimeAction: Date;
    commands: ActionPointAtTimeModel[];

    public constructor(_hourTimeAction: Date, _commands: ActionPointAtTimeModel[]){
        this.hourTimeAction = _hourTimeAction;
        this.commands = _commands;
    }
}